src/app/app.routing.ts

```diff
@@ -7,6 +7,7 @@ import { CommonLayoutComponent } from './layouts/common-layout/common-layout.com
 import { PasswordResetComponent } from './pages/password-reset/password-reset.component';
 import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-reset-password.component';
 import { NoAuthGuard } from './guards/no-auth.guard';
+import { PaymentsComponent } from "./pages/internationalPayment/payments.component";
```

Comment:
Hey @patrickf949, please run `npm run lint` to see and fix all the linting issues in your code.